src/ports/postgres/modules/deep_learning/input_data_preprocessor.sql_in

```diff
@@ -849,3 +857,13 @@ import numpy as np
 return np.frombuffer(var, dtype=np.float32)
```

Comment:
I must have missed this last time, but shouldn't this be `tobytes()` instead of `frombuffer()`?  (For function `convert_array_to_bytea()`)

Also, maybe we should rename it to `convert_real_array_to_bytea()` now that we have two different array types.